devel/mercurial/Portfile

```diff
@@ -7,7 +7,7 @@ PortGroup           bitbucket 1.0
 name                mercurial
 # don't forget to update dependents for mercurial:
```

Comment:
Please read the comments in the file, just one line above the changed version:
```
# don't forget to update dependents for mercurial:
# port echo rdepends:mercurial and \( name:hg or name:mercurial \) | grep -v 'py[[:digit:]]'
```
While I didn't test yet, I'm pretty sure the dependencies are broken.

Reply:
The only bad news is that TortoiseHG only "released" 5.3rc0 so far, but I guess I would go with that one, else the software is useless.

Reply:
Which software is useless?

The other `rdepends` install cleanly here; I'm not clear on what the `tortoisehg` problem is since I've never used it and to test I'd have to install Qt5.

Reply:
@0branch: `tortoisehg` may install cleanly (I didn't test), but it refuses to even start. I upgraded the port to a RC version in the meantime.

Reply:
Thanks.